maven-toolchain-builder/src/main/java/org/apache/maven/toolchain/discovery/JavaHomeFinder.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
The PR says "This borrows code from IntelliJ Idea (which is ASL2)" and I guess this is one of the classes that was borrowed from there (given the comment in line 44). Does this text in the license header still make sense then, or should it reflect the original code that we borrow?

Reply:
You're right,  the files are licensed under ASL2, but the copyright should be maintained.  I'll add it back.